inst/qml/AnovaBayesian.qml

```diff
@@ -16,10 +16,9 @@
 // <http://www.gnu.org/licenses/>.
 //
 
-import QtQuick			2.12
-import JASP.Controls	1.0
-import JASP.Widgets		1.0
-import JASP				1.0
```

Comment:
removed `JASP.Widgets`

Reply:
Yes JASP.Widgets should not be used by the Analysis Forms anymore. JASP.Controls should contain all controls needed for the forms. JASP.Widgets is for extra widgets used by JASP Desktop.